juju/juju
# openstack storage provider now working with CACerts

## Description of change

Make use of the cinder client change to allow for tls.Config data to be used when making api calls to an OpenStack requiring CACert as part of authentication.

## QA steps

Using an openstack deployment requiring a CACert for access:
1. Add the ca cert to the clouds.yaml for the new cloud:
clouds:
openstack:
type: openstack
.....
ca-certificates:
|
-----BEGIN CERTIFICATE-----
.....
-----END CERTIFICATE-----
1. juju bootstrap
2. use juju storage functionality.
3. juju destroy-controller should not fail with unable to list volumes.

There should be no change with an openstack deployment not requiring a CACert.

## Documentation changes

Not yet, one more item to be fixed.

## Bug reference

https://bugs.launchpad.net/juju/+bug/1784030
